DNN Platform/Library/Entities/Tabs/TabController.cs

```diff
@@ -1704,6 +1712,10 @@ public void MoveTabToParent(TabInfo tab, int parentId)
 
             //Clear the Cache
             ClearCache(tab.PortalID);
+
+            var portalId = GetPortalId(tab.TabID, -1);

```

Comment:
Please use  PortalController.Instance.GetCurrentPortalSettings() to get portal ID rather than -1 in above changes as well.

Reply:
IMO -1 is ok here, as there have other places used -1 as well. shoudn't use GetCurrentPortalSettings as it will be NULL if called from scheduler.